index.js

```diff
@@ -34,6 +34,7 @@ function Polyglot(options) {
   this.phrases = {};
   this.extend(opts.phrases || {});
   this.currentLocale = opts.locale || 'en';
+  this.numberFormat = opts.numberFormat || { format: String };
```

Comment:
i feel like it'd be better to just accept a function here, and let the user handle preserving the receiver if desired. ie, the default here should maybe just be `String`?